starter-code/iteration-2/main.ts

```diff
@@ -1,6 +1,36 @@
 // Add the reference to the "TodoInterface"
-
+interface TodoInterface {
+  myArray: Array<string>;
+  addTask(string: string):number;
+  listAllTasks():void;
+  deleteTask(string: string): number;
+}
 // 1. Create a class Todo that implements the Interface created before.
+class Todo implements TodoInterface {
+ 
+  constructor(public myArray: Array<string>){}
```

Comment:
```
class Todo implements TodoInterface {
    taskArray: Array<string> = [];
    constructor() { }

    addTask(task): number {
        this.taskArray.push(task);
        console.log(`=========== NEW TASK ===========
            Task ${task} inserted in the list
            Number of items: ${this.taskArray.length}`);
        return this.taskArray.length;
    }
    ...
}
```

Reply:
You have to give an empty value to the string if you want to push it.

Reply:
Here you are asking for an argument in the constructor:
```constructor (public taskArray : Array <string>) ```
When you call this constructor you are not passing any argument:
``` let myTodos = new Todo();```